pkgs/by-name/fr/freediameter/package.nix

```
@@ -0,0 +1,61 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  pkg-config,
+  flex,
+  bison,
+  lksctp-tools,
+  libidn2,
+  libgcrypt,
+  gnutls,
+  fetchFromGitHub,
+  unstableGitUpdater,
```

Bit of a nit, but let's sort our inputs in the following order: builder, build-time deps, run-time deps, update script

```suggestion
  lib,
  stdenv,
  fetchFromGitHub,
  bison,
  cmake,
  flex,
  pkg-config,
  gnutls,
  libgcrypt,
  libidn2,
  lksctp-tools,
  unstableGitUpdater,
```